ArduCopter/defines.h

```diff
@@ -160,3 +160,11 @@ enum LoggingParameters {
 #define THR_BEHAVE_FEEDBACK_FROM_MID_STICK (1<<0)
 #define THR_BEHAVE_HIGH_THROTTLE_CANCELS_LAND (1<<1)
 #define THR_BEHAVE_DISARM_ON_LAND_DETECT (1<<2)
+
+// FLIP_MODE parameters
+enum FlipModeDir{
```

Comment:
```suggestion
enum class FlipModeDir {
```

Reply:
could this enum live in mode flip? I don't think we need it in the copter namespace?

Reply:
We could move the enum to the ModeFlip.cpp file, the only fallout I can foresee is that when defining the default parameters in Parameters.cpp we will have to use the integer values instead of the enums.

Reply:
You should still be able to get at them in the mode flip header (in `mode.h`) EG `ModeFlip::FlipModeDir::FLIP_LEFT` In that case you could actuialy shorten the enum name. `ModeFlip::Dir::LEFT` don't need the mode and flip in there multiple times.